controllers/nutanixmachine_controller.go

```diff
@@ -54,8 +55,23 @@ const (
 	// provideridFmt is "nutanix://<vmUUID"
 	provideridFmt = "nutanix://%s"
 	projectKind   = "project"
+	ONE_KIB       = 1024
+	ONE_MIB       = ONE_KIB * ONE_KIB
+	ONE_GIB       = ONE_MIB * ONE_KIB
 )
 
+var (
+	minMachineSystemDiskSize *resource.Quantity
+	minMachineMemorySize     *resource.Quantity
+	minVCPUsPerSocket        = 1
+	minVCPUSockets           = 1
+)
+
+func init() {
+	minMachineSystemDiskSize = resource.NewQuantity(20*ONE_GIB, resource.BinarySI)
```

Comment:
You can just do resource.MustParse here instead of defining KiB, MiB, and GiB manually. https://pkg.go.dev/k8s.io/apimachinery/pkg/api/resource#example-MustParse